core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -672,8 +719,10 @@ class EntityApiServiceSpec extends ApiServiceSpec {
           assertResult(StatusCodes.NoContent) {
             status
           }
-          assertResult(Some(new_e1)) {
-            runAndWait(entityQuery.get(testData.workspace, e1.entityType, e1.name))
+          assertResult(new_e1) {
+            runAndWait(
+              compactEntityRepository.queries.getEntity(testData.workspace.workspaceIdAsUUID, e1.entityType, e1.name)
+            ).get.toEntity
```

Comment:
no need to change this, but just for discussion: you could leave the expected value as-is and use `.map` instead of `.get`. In the case of a failure where the entity is not found, this would output an error message showing the expected Some(new_e1) and actual None values instead of throwing an error on invocation of `.get`.
```scala
          assertResult(Some(new_e1)) {
            runAndWait(
              compactEntityRepository.queries.getEntity(testData.workspace.workspaceIdAsUUID, e1.entityType, e1.name)
            ).map(_.toEntity)
          }
```

Reply:
That is better though